projects/canopy/src/lib/hero-img/hero-img.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'lg-hero-img',
+  templateUrl: './hero-img.component.html',
+  styleUrls: ['./hero-img.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
We tend to use `changeDetection: ChangeDetectionStrategy.OnPush` on our parent components - sub components not so much, are you okay to add that here?